example/spec/parallel/testAPI/granuleSpec.js

```diff
@@ -81,11 +83,31 @@ describe('The Granules API', () => {
     });
 
     it('can discover the granule in the API.', async () => {
-      const granule = await getGranule({
+      discoveredGranule = await getGranule({
         prefix,
         granuleId,
       });
-      expect(granule).toEqual(jasmine.objectContaining(randomGranuleRecord));
+      expect(discoveredGranule).toEqual(jasmine.objectContaining(randomGranuleRecord));
+    });
+
+    it('can modify the granule via API.', async () => {
+      const modifiedGranule = {
+        ...discoveredGranule,
+        status: 'failed',
+        error: { message: 'granule now failed' },
+      };
+      const response = await updateGranule({
+        prefix,
+        body: modifiedGranule,
+      });
+
+      expect(response.statusCode).toBe(200);
+      const updatedGranuleFromApi = await getGranule({
+        prefix,
+        granuleId: modifiedGranule.granuleId,
+      });
+      updatedGranuleFromApi.execution = undefined;
+      expect(updatedGranuleFromApi).toEqual(jasmine.objectContaining(modifiedGranule));
     });
 
```

Comment:
suggest to add a test to `search` the granule to make sure the granule is added to `es`

Reply:
0bd7337